sdk/src/test/resources/data/objects/source_object.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think we should have fixtures files for each source types

Reply:
The serialization test is broken anyway, I'll need to revamp it in the testing task.

Reply:
You mean you need to do it in another PR?

Reply:
Yeah there was already a task to complete testing for Requests. During this task I realized that the serialization test was actually broken so I add this task -> **T14297** as a subtask of the testing task.